nav2_behavior_tree/plugins/condition/goal_updated_condition.cpp

```diff
@@ -0,0 +1,76 @@
+#ifndef NAV2_BEHAVIOR_TREE__GOAL_UPDATED_CONDITION_HPP_
```

Comment:
I am missing the copyright here, because I don't really know what I am supposed to write :)
Any ideas?

Reply:
Copy paste from another file and just change to your name 

(literally)

Reply:
Perfect, thanks!